addons_engine/README.md

```diff
@@ -0,0 +1,14 @@
+# Addon Engine
+
+
+## Ingoing Endpoints
+
+
+For more details checkout out the [openapi.json](./api/v1/openapi.json)
+
+
+## Start Service
+
+Run the service by running `./start-up.sh`
```

Comment:
I get the same error:
![image](https://github.com/oakestra/oakestra/assets/65814168/8d922f3f-f4a2-49fd-b839-d0619716f0e0)

Even if I install vevns like this: sudo apt install python3-virtualenv

Reply:
Do any start-up.sh in the project work for you?

Reply:
I quickly searched the error you reported. Is it possible you don't have python3.8 installed in your machine? If so try modifiying the start-up.sh line to be:
`virtualenv --clear -p python3 .venv` (python3 instead of python3.8)

Reply:
It works when I change pyhton3.8 to python3.
It would be cool if you could change that so that these scripts are not hardcoded to version 3.8 (I am using 3.10 and up)